gh-pages/_sass/layout.scss

```diff
@@ -93,12 +93,17 @@ $site-nav-width: 300px;
 
 .page-content {
 	background: white;
-	margin: 90px 0 40px 0;
+	margin: 45px 0 40px 0;
```

Comment:
I deployed this branch to wikia.github.io/style-guide and I see now there is very little margin above the content on pages which aren't the home page. Can that be adjusted?